Bugzilla/Install/Filesystem.pm

```diff
@@ -31,6 +31,7 @@ use File::Path;
 use File::Basename;
 use File::Copy qw(move);
 use File::Spec;
+use File::stat;
```

Comment:
Seriously, they made this *lowercase*? Augh. Unless it's a typo. Probably isn't!